### ISSUE TITLE
Use of undefined constant PIWIK_INCLUDE_PATH

### ISSUE THREAD

**Issue by idevwebs** [matomo-org/matomo#14346]
I had this installed and working before. Not sure what is causing this error. I tried deleting the entire matomo folder and replacing from fresh zip. 

```
Fatal error: Uncaught Error: Class 'Piwik\SettingsServer' not found in /home/site/public_html/matomo/core/testMinimumPhpVersion.php:87 Stack trace: 
#0 /home/site/public_html/matomo/core/testMinimumPhpVersion.php(175): Piwik_ShouldPrintBackTraceWithMessage() 
#1 /home/site/public_html/matomo/core/testMinimumPhpVersion.php(194): Piwik_GetErrorMessagePage('<p><strong>Mato...', false, '<ul>\n ...') 
#2 /home/site/public_html/matomo/core/bootstrap.php(30): require_once('/home/site/p...') 
#3 /home/site/public_html/matomo/index.php(21): require_once('/home/site/p...') 
#4 {main} thrown in /home/site/public_html/matomo/core/testMinimumPhpVersion.php on line 87
```

**Comment by idevwebs** [comment]
Additional info:
php 7.3

```
[16-Apr-2019 05:23:55 UTC] PHP Warning:  Use of undefined constant PIWIK_INCLUDE_PATH - assumed 'PIWIK_INCLUDE_PATH' (this will throw an Error in a future version of PHP) in /home/site/public_html/matomo/core/bootstrap.php on line 30
[16-Apr-2019 05:23:55 UTC] PHP Warning:  require_once(PIWIK_INCLUDE_PATH/core/testMinimumPhpVersion.php): failed to open stream: No such file or directory in /home/site/public_html/matomo/core/bootstrap.php on line 30
[16-Apr-2019 05:23:55 UTC] PHP Fatal error:  require_once(): Failed opening required 'PIWIK_INCLUDE_PATH/core/testMinimumPhpVersion.php' (include_path='.:/opt/cpanel/ea-php73/root/usr/share/pear') in /home/site/public_html/matomo/core/bootstrap.php on line 30
```

**Comment by fdellwing** [comment]
Can you post the first ~20 lines from `piwik.php` and `index.php`?

**Comment by tsteur** [comment]
Hi there,

it sounds like some file wasn't copied properly during the update. I recommend you update the files again using the manual update see https://matomo.org/docs/update/#the-manual-three-step-update

Make sure to backup your `config/config.ini.php` and do not delete it as mentioned in the docs. This should fix your issue. Let us know if this doesn't work and we're happy to look at it again and reopen the issue.

**Comment by idevwebs** [comment]
FYI this turned out to be a php `include_path` config issue.

**Comment by acch** [comment]
> FYI this turned out to be a php `include_path` config issue.

Would you mind letting me know how you've solved this? Looks like I'm facing the same / similar issue... Thanks!

**Comment by fdellwing** [comment]
I read, that `set_include_path()` may fail if you setup an include path in your `php.ini`. So, you could check if that is the case for you and if yes, comment it out and test it again?

**Comment by idevwebs** [comment]
I don't recall everything, but here's what I do recall:

1. I use WHM and in efforts to secure things, I had configured php open_basedir to be limited to user home and in my configuration, I had to add another value to set include_path to start there as well as setting upload_tmp_dir. Those changes were done in system_pool_defaults.yaml
2. Eventually I enabled ini_set after numerous other security things were done first.
3. I initially modified the matomo/core/testMinimumVersion.php commenting the exit(1) at the end before doing #1 and #2 above.
